VATSpy.dat

```diff
@@ -17940,6 +17940,7 @@ LGMD-T|Makedonia-(TSL sector)|LGMD-T|LGMD-T
 LHCC|Budapest||LHCC
 LIBB|Brindisi||LIBB
 LIMM|Milano||LIMM
+LIPP|Padova|LIMM|LIMM

```

Comment:
This would cause LIMM sector to "light up" whenever LIPP was online as well, which is not correct as LIMM covers LIPP when this one is not online, but not the other way around. My recommendation would be instead of creating a new line, simply adding "LIMM" as an alias in the already existing one (just below this one), so it ends up being `LIPP|Padova|LIMM|LIPP`.

Reply:
fixed thanks for letting me know about this @prryplatypus